docs/source/quickstart.md

```diff
@@ -31,27 +31,27 @@ A few other goals:
 
 ## Main concepts
 
-The library is build around three type of classes for each models:
+The library is build around three types of classes for each model:
 
-- **model classes** which are PyTorch models (`torch.nn.Modules`) of the 8 models architectures currently provided in the library, e.g. `BertModel`
-- **configuration classes** which store all the parameters required to build a model, e.g. `BertConfig`. You don't always need to instantiate these your-self, in particular if you are using a pretrained model without any modification, creating the model will automatically take care of instantiating the configuration (which is part of the model)
-- **tokenizer classes** which store the vocabulary for each model and provide methods for encoding/decoding strings in list of token embeddings indices to be fed to a model, e.g. `BertTokenizer`
+- **model classes** which are PyTorch models (`torch.nn.Modules`) of the 8 models architectures currently provided in the library, e.g., `BertModel`
```

Comment:
there are more than 8 architectures I think

Reply:
I am not familiar with the complete library. So, I (focused on and) have corrected only grammatical mistakes.